src/test/java/org/assertj/core/presentation/StandardRepresentation_unambiguousToStringOf_Test.java

```diff
@@ -0,0 +1,431 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2018 the original author or authors.
+ */
+package org.assertj.core.presentation;
+
+import static java.lang.Integer.toHexString;
+import static java.lang.String.format;
+import static java.util.concurrent.atomic.AtomicReferenceFieldUpdater.newUpdater;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.entry;
+import static org.assertj.core.api.Assertions.tuple;
+import static org.assertj.core.util.Arrays.array;
+import static org.assertj.core.util.Lists.newArrayList;
+
+import java.io.File;
+import java.lang.reflect.Method;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Collection;
+import java.util.Comparator;
+import java.util.Date;
+import java.util.GregorianCalendar;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicIntegerFieldUpdater;
+import java.util.concurrent.atomic.AtomicLongFieldUpdater;
+import java.util.concurrent.atomic.AtomicMarkableReference;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.concurrent.atomic.AtomicReferenceFieldUpdater;
+import java.util.concurrent.atomic.AtomicStampedReference;
+
+import org.assertj.core.data.MapEntry;
+import org.assertj.core.groups.Tuple;
+import org.assertj.core.util.OtherStringTestComparator;
+import org.assertj.core.util.OtherStringTestComparatorWithAt;
+import org.assertj.core.util.StringTestComparator;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Tests for {@link StandardRepresentation#unambiguousToStringOf(Object)}.
+ *
+ * @author Alexandre Dutra
+ */
+public class StandardRepresentation_unambiguousToStringOf_Test extends AbstractBaseRepresentationTest {
+
+  private static final StandardRepresentation STANDARD_REPRESENTATION = new StandardRepresentation();
+
+  @Test
+  public void should_return_null_if_object_is_null() {
+    assertThat(unambiguousToStringOf(null)).isNull();
+  }
+
+  @Test
+  public void should_quote_String() {
+    String obj = "Hello";
+    assertThat(unambiguousToStringOf(obj)).isEqualTo(format("\"Hello\" (String@%s)",
```

Comment:
I'm using IntelliJ with the Eclipse formatter profile as indicated in the contribution guidelines, and this is how the code gets formatted. Not sure if this is 100% accurate.

Reply:
don't worry about formatting too much, I usually end up fixing it anyway.